### PR TITLE
Docs: Update schedules.md for deployments from cli and python file

### PR DESCRIPTION
**Pull request by discdiver**
Improve docs Schedules concept page to show new common ways of creating schedules.

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a `fix`, `feature`, `enhancement`, `docs`, or `maintenance` label categorizing the change.

### CODE REVIEW

File: docs/concepts/schedules.md [hunk 1]
```diff
@@ -16,6 +16,13 @@ Schedules tell the Prefect API how to create new flow runs for you automatically
 
 You can add a schedule to any flow [deployment](/concepts/deployments/). The Prefect `Scheduler` service periodically reviews every deployment and creates new flow runs according to the schedule configured for the deployment.
 
+There are four recommended possible ways to create a schedule for a deployment:
```
Comment (cicdw) [inline docs/concepts/schedules.md:19]:
```suggestion
There are four recommended ways to create a schedule for a deployment:
```

Comment (cicdw) [inline docs/concepts/schedules.md:19]:
```suggestion
There are four recommended ways to create a schedule for a deployment:
```

File: docs/concepts/schedules.md [hunk 2]
```diff
@@ -16,6 +16,13 @@ Schedules tell the Prefect API how to create new flow runs for you automatically
...
+
+- Use the Prefect UI
+- Use the `--schedule` flag with the CLI `deployment build` command
```
Comment (cicdw) [inline docs/concepts/schedules.md:22]: Unfortunately we don't support a single `--schedule` flag, there are 3 separate flags, one for each schedule type; not sure of the best way to say that here.

File: docs/concepts/schedules.md [hunk 3]
```diff
@@ -16,6 +16,13 @@ Schedules tell the Prefect API how to create new flow runs for you automatically
...
+- Use the `schedule` parameter with a Python deployment file
+- Edit the deployment YAML file's `schedule` section
```
Comment (cicdw) [inline docs/concepts/schedules.md:24]:
```suggestion
- Manually edit the deployment YAML file's `schedule` section
```

File: docs/concepts/schedules.md [hunk 4]
```diff
@@ -39,17 +46,47 @@ Prefect supports several types of schedules that cover a wide range of use cases
...
+
+When you build a deployment from the CLI you can use `cron`, `interval`, or `rrule` flags to set a schedule. 
+
+<div class="terminal">
+```bash
+prefect deployment build demo.py:pipeline -n etl --cron "0 0 * * *"
+```
+</div>
+
+This schedule will set flow runs in a deployment every day at midnight. You cannot set a timezone for your schedule from the CLI at the present time.
```
Comment (cicdw) [inline docs/concepts/schedules.md:59]:
```suggestion
This schedule will create flow runs for this deployment every day at midnight. You cannot set a timezone for your schedule from the CLI at the present time.
```

**Review by cicdw — CHANGES_REQUESTED** [review]
a few minor wording suggestions

File: docs/concepts/schedules.md (hunk 2 above)
Comment (discdiver) [reply docs/concepts/schedules.md:22]: Good catch.  I'll change to "Use the `cron`, `interval`, or `rrule` flags with the CLI  `deployment build` command.

File: docs/concepts/schedules.md [hunk 5]
```diff
@@ -16,6 +16,13 @@ Schedules tell the Prefect API how to create new flow runs for you automatically
 
 You can add a schedule to any flow [deployment](/concepts/deployments/). The Prefect `Scheduler` service periodically reviews every deployment and creates new flow runs according to the schedule configured for the deployment.
 
+There are four recommended ways to create a schedule for a deployment:
+
+- Use the Prefect UI
+- Use the `--schedule` flag with the CLI `deployment build` command
```
Comment (discdiver) [inline docs/concepts/schedules.md:22]:
```suggestion
- "Use the `cron`, `interval`, or `rrule` flags with the CLI  `deployment build` command
```

File: docs/concepts/schedules.md [hunk 6]
```diff
@@ -16,6 +16,13 @@ Schedules tell the Prefect API how to create new flow runs for you automatically
 
 You can add a schedule to any flow [deployment](/concepts/deployments/). The Prefect `Scheduler` service periodically reviews every deployment and creates new flow runs according to the schedule configured for the deployment.
 
+There are four recommended ways to create a schedule for a deployment:
+
+- Use the Prefect UI
+- "Use the `cron`, `interval`, or `rrule` flags with the CLI  `deployment build` command
```
Comment (cicdw) [inline docs/concepts/schedules.md:22]:
```suggestion
- Use the `cron`, `interval`, or `rrule` flags with the CLI  `deployment build` command
```
accidental double quote

**Review by cicdw — APPROVED** [review]

**Pull request merged**